cscs-checks/prgenv/gpu_direct_acc.py

```diff
@@ -39,8 +38,12 @@ def __init__(self, **kwargs):
 
     def setup(self, partition, environ, **job_opts):
         if environ.name.startswith('PrgEnv-cray'):
-            environ.fflags = '-hacc -hnoomp'
+            self.build_system.fflags = ['-hacc', '-hnoomp']
         elif environ.name.startswith('PrgEnv-pgi'):
-            environ.fflags = self._pgi_flags
+            if self.current_system.name in ['daint', 'dom']:
+                self.build_system.fflags = ['-acc', '-ta=tesla:cc60',
+                                            '-Mnorpath']
```

Comment:
I would move the `self.build_system.fflags = ['-acc']` outside the `if` and here use `+=`